### ISSUE TITLE
Particles

### ISSUE THREAD

**Issue by TailsDollLoco** [NyaaCat/RPGItems-reloaded#288]
How i put more particles to the item's? o are a plugin to put other particles...
And how i shoot particles?

**Comment by phoenixlzx** [comment]
One RPGitem can have one particle at a time.

I think we need a power to shoot particles ... leeder just told me last night. @Librazy what do you think?

**Comment by sirjavlux** [comment]
i think it would be a mayor thing to be able to make wands that shoots particles that deals damage on hit or somthing like that.

**Comment by Midnight9746** [comment]
@phoenixlzx what about a particle that's triggered after a condition? Like if a player gets hit, it spawns say like witch magic. If the player (of the item) dies, it spawns like a puff of smoke. If someone else dies to the item, (victims) it does it's own particles.

Also be cool to see rpg sets where you need the full set to unlock extra powers.

Like /rpgitem [item] requires [item2] | [item3}
Then /rpgitem [item] required [power/particle]

In other words, think about King Author's sword. Alone it does nothing, but wearing the suit of armor and the sword goes from like 15 damamge to 30, and the player gets a particle effect. :D
